### PR TITLE
1790 api

### PR DESCRIPTION
**Pull request by b-gehrke**
haskell hets API with first steps for Python API

### CODE REVIEW

**Comment by b-gehrke** [comment]
Remove old Changes and see if still works

File: Hets.hs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module Hets (
```
Comment (b-gehrke) [inline Hets.hs:1]: Rename to HetsAPI

File: Hets/Commands.hs [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+libFlatHiding = globalCommandR Hiding
+
+libFlatHeterogen :: LibName -> LibEnv -> Result LibEnv
+libFlatHeterogen = globalCommandR Heterogeneity
+
+qualifyLibEnv :: LibName -> LibEnv -> Result LibEnv
+qualifyLibEnv = globalCommandR QualifyNames
+
+
+loadLibrary :: FilePath -> HetcatsOpts -> IO (Result (LibName, LibEnv))
```
Comment (b-gehrke) [inline Hets/Commands.hs:143]: Unify with read and analyse

File: Hets/Python.hs [hunk 3]
```diff
@@ -0,0 +1,126 @@
+module Hets.Python (
```
Comment (b-gehrke) [inline Hets/Python.hs:1]: For G_theory show sign as set as strings

Comment (b-gehrke) [reply Hets/Python.hs:1]: use `Logic.Logic.sym_of` and show the symbols

File: Docker/hyphen.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
+FROM ubuntu:22.04
```
Comment (b-gehrke) [inline Docker/hyphen.Dockerfile:2]: Change to :lts

File: Setup.hs [hunk 5]
```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
```
Comment (tillmo) [inline Setup.hs:1]: what purpose does this file have? Please add a comment, or remove the file

File: Driver/ReadFn.hs [hunk 6]
```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP #-}
```
[the remark this answers is not in the document]
Comment (b-gehrke) [reply Driver/ReadFn.hs]: It's not. Removed it.

File: Setup.hs (hunk 5 above)
Comment (b-gehrke) [reply Setup.hs:4]: Comment added. This file is only used for library building. Cabal generates this file by default when creating a new library project.

**Review by tillmo — APPROVED** [review]

**Pull request merged**